extensions/adobe/experience/profile/profile-all.schema.json

```diff
@@ -66,6 +66,25 @@
               }
             }
           }
+        },
+        "https://ns.adobe.com/experience/profile/consentPolicies": {
+          "title": "Consent policies map",
+          "type": "object",
+          "meta:xdmType": "map",
+          "additionalProperties": {
+            "title": "consent policy",
+            "type": "array",
+            "items": {
+              "type": "object",
```

Comment:
I don't see why this is an object with a single field, "id"? I think it would be more efficient to simply make this an array of strings.

Reply:
If we don't anticipate any more fields to be added in the future, then I can make it a simple list of strings.